AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DepthOfFieldAdded.py

```diff
@@ -30,15 +31,141 @@ class Tests:
     depth_of_field_component = (
         "Entity has a DepthOfField component",
         "Entity failed to find DepthOfField component")
-    depth_of_field_disabled = (
+    depth_of_field_component_disabled = (
         "DepthOfField component disabled",
-        "DepthOfField component was not disabled.")
+        "DepthOfField component was not disabled")
```

Comment:
please mark all fail strings with P0: for existing Tests tuple and P1: for the new tests covered under this work

Reply:
fixed